scalableminds/webknossos
# Refactor mapping related model code

I've moved the mapping related code from the binary.js to the mappings module. I've also deduplicated some functionality/properties (isMappingEnabled, tracing flow types) and refactored other minor stuff that I saw while doing this.
This PR also removes API v1 as it is obsolete, the tests that previously tested v1 have been adapted to test v2.

### Steps to test:
- Open a volume tracing
- Use our API to activate a mapping `webknossos.apiReady(2).then( api => api.data.setMapping(api.data.getVolumeTracingLayerName(), {1230625: 0}))` with the id of the cell at the current position => The color should disappear
- Use the `mappings` object on the ROI2017_wkw dataset and activate a predefined mapping `mapping.activate("axons")` => The mapping should be activated
- Open the mappings tab and disable/enable the mapping

### Issues:
- part of #2601

------
